src/math/Vector4.js

```diff
@@ -413,6 +413,19 @@ class Vector4 {
 
 	}
 
+	setFromMatrixPosition( m ) {
```

Comment:
Do you mind documenting this method here?

https://threejs.org/docs/index.html#api/en/math/Vector4

You can use the docs from the [Vector3](https://threejs.org/docs/index.html#api/en/math/Vector3.setFromMatrixPosition) version as a tempalte.

Reply:
Ah, missed that. Thanks!